lektor/i18n.py

```diff
@@ -13,7 +14,8 @@
 
 translations = {}
 for _lang in KNOWN_LANGUAGES:
-    with open(os.path.join(translations_path, _lang + '.json')) as f:
+    with open(os.path.join(translations_path, _lang + '.json'),
+              encoding="utf8") as f:
```

Comment:
Also it would be nice to add a comment to the issues that generated the changed abothe the `with io.open`

# See https://github.com/lektor/lektor/issues/378